src/main/java/org/codehaus/groovy/ast/expr/RangeExpression.java

```diff
@@ -64,10 +80,20 @@ public boolean isInclusive() {
         return inclusive;
     }
 
+    public boolean isExclusiveLeft() {
+        return exclusiveLeft;
+    }
+
+    public boolean isExclusiveRight() {
+        return exclusiveRight;
+    }
+
     @Override
```

Comment:
Would "isLeftExclusive" or "isLeftOpen" be better than "isExclusiveLeft"?  And same for right?

Reply:
I agree that it would be clearer to use "open" keyword instead of "exclusive". `isOpenLeft` and `isOpenRight` are more similiar to each other than `isLeftOpen` and `isRightOpen` so it think it would be better to go with that way. IDE autocompletion will also benefit from this, e.g. writing `isOpen...` will pop up both.

Reply:
On second thought, "exclusive" keyword is widely used elsewhere, so keeping the names as they are now would keep things more consistent.